pkg/system/phase4_configuring.go

```diff
@@ -747,6 +790,23 @@ func (r *Reconciler) ReconcileDefaultBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) prepareDefaultNsfsPvc() error {
+	// We need to create a PVC with specturm scale storageclass. nsfs-local-test is just for testing.
+	var sc = "ibm-spectrum-scale-csi-storageclass-version2"
+	defaultPVCSize := int64(16) * 1024 * 1024 * 1024 // 16GB
```

Comment:
16GB is too low, Let's use 30GB like we did for pv-pool.

Reply:
done.